src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -249,6 +253,15 @@ class ExternalIssueActions extends AsyncComponent {
           externalIssueKey={issue ? issue.key : null}
           onClose={this.deleteIssue.bind(this)}
           integrationType={selectedIntegration.provider.key}
+          hoverCardHeader={`Integration: ${selectedIntegration.provider.name}`}
+          hoverCardBody={
+            (issue && issue.title) ?
+            (<div>
+              <IssueTitle>{issue.title}</IssueTitle>
+              {issue.description && <IssueDescription>{issue.description}</IssueDescription>}
+            </div>) :
+            <IntegrationItem integration={selectedIntegration} compact={false} />
```

Comment:
Isn't it by default compact?

Reply:
looks like the current default is actually `expanded`

Reply:
er...there is no default be the ternary that decides the layout falls back to expanded if null is passed

Reply:
Sorry my comment here was wrong, I meant doesn't it by default NOT use compact?

Reply:
lol WHY THEN YES